src/apps/search-header/helpers.ts

```diff
@@ -5,6 +5,13 @@ import {
   SuggestionsFromQueryStringQuery,
   SuggestionTypeEnum
 } from "../../core/dbc-gateway/generated/graphql";
+import {
```

Comment:
I wonder if it makes sense to have this functionality in the serach-header app alone?
It seems like it is fundamental search logic that should be part of a global search engine.
I don't recall where the rest of the seach logic is placed , though 🤔